server/api/store.go

```diff
@@ -365,7 +378,19 @@ func (h *storesHandler) SetAllLimit(w http.ResponseWriter, r *http.Request) {
 }
 
 func (h *storesHandler) GetAllLimit(w http.ResponseWriter, r *http.Request) {
-	limits, err := h.GetAllStoresLimit()
+	typeName := r.URL.Query().Get("type")
+	var typeValue storelimit.Type
+	if typeName == "" {
```

Comment:
BTW, we can set a default value for `typeValue` to avoid some if else

Reply:
Done